### PR TITLE
added alt text for embedded videos in the docs

### PR DESCRIPTION
**Pull request by shikhar13012001**
#### Summary

added alt text for embedded videos in the docs. Please review my pr and suggest if you need any changes, thank you for assigning this issue to me.

### CODE REVIEW

**Comment by shikhar13012001** [comment]
Hi @cwarnermm, Can you please review my PR? Thanks !

**Comment by justinegeffen** [comment]
Thanks for your PR, @shikhar13012001! We're in the process of reviewing Hacktoberfest PRs. Your contribution has been acknowledged as a Hacktoberfest PR. Thanks for your patience! :)

File: source/archive/organizing-conversations-copy2.rst [hunk 1]
```diff
@@ -92,5 +92,5 @@ For additional details about working with the Beta release of Collapsed Reply Th
 .. raw:: html
 
    <div style="position: relative; padding-bottom: 50%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
-      <iframe src="https://www.youtube.com/embed/hnD0Zj-mIbs" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>
+      <iframe src="https://www.youtube.com/embed/hnD0Zj-mIbs" frameborder="0" allowfullscreen alt="video on organizing converstations copy 2" style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>
```
Comment (cwarnermm) [inline source/archive/organizing-conversations-copy2.rst:95]:
```suggestion
      <iframe src="https://www.youtube.com/embed/hnD0Zj-mIbs" frameborder="0" allowfullscreen alt="Video on organizing Mattermost conversations" style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>
```

File: source/archive/organizing-conversations.rst [hunk 2]
```diff
@@ -75,5 +75,5 @@ For additional details about working with the Beta release of Collapsed Reply Th
 .. raw:: html
 
    <div style="position: relative; padding-bottom: 50%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
-      <iframe src="https://www.youtube.com/embed/hnD0Zj-mIbs" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>
+      <iframe src="https://www.youtube.com/embed/hnD0Zj-mIbs" alt="video on organizing converstion" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>
```
Comment (cwarnermm) [inline source/archive/organizing-conversations.rst:78]: @shikhar13012001 - Source files located in the ``archive`` folder are no longer built into the product documentation and shouldn't be updated. Please revert this change.

File: source/manage/mmctl-command-line-tool.rst [hunk 3]
```diff
@@ -24,7 +24,7 @@ For more information on what's new in mmctl for Mattermost v6.0, see the `Matter
 .. raw:: html
 
    <div style="position: relative; padding-bottom: 50%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
-      <iframe src="https://www.youtube.com/embed/zITNaPWP-80" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>
+      <iframe src="https://www.youtube.com/embed/zITNaPWP-80" alt="video on usage of mmctl command line tool" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>
```
Comment (cwarnermm) [inline source/manage/mmctl-command-line-tool.rst:27]:
```suggestion
      <iframe src="https://www.youtube.com/embed/zITNaPWP-80" alt="Video on what's new in the mmctl command line tool from Mattermost v6.0" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>
```

File: source/manage/mmctl-command-line-tool.rst [hunk 4]
```diff
@@ -41,7 +41,7 @@ For more information on what's changed in mmctl for Mattermost v6.0, see the `Ma
 .. raw:: html
 
    <div style="position: relative; padding-bottom: 50%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
-      <iframe src="https://www.youtube.com/embed/hmbSfSeWo4Y" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>
+      <iframe src="https://www.youtube.com/embed/hmbSfSeWo4Y" alt="video for more information on what's changed in mmctl for Mattermost v6.0" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>
```
Comment (cwarnermm) [inline source/manage/mmctl-command-line-tool.rst:44]:
```suggestion
      <iframe src="https://www.youtube.com/embed/hmbSfSeWo4Y" alt="Video for more information on what's changed in mmctl for Mattermost v6.0" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>
```

File: source/onboard/ad-ldap-groups-synchronization.rst [hunk 5]
```diff
@@ -25,7 +25,7 @@ You can also watch a video overview about adding users to Mattermost with AD/LDA
 .. raw:: html
 
    <div style="position: relative; padding-bottom: 50%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
-      <iframe src="https://www.youtube.com/embed/zyku2ibsG0M" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>
+      <iframe src="https://www.youtube.com/embed/zyku2ibsG0M" alt="video overview about adding users to Mattermost with AD/LDAP" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>
```
Comment (cwarnermm) [inline source/onboard/ad-ldap-groups-synchronization.rst:28]:
```suggestion
      <iframe src="https://www.youtube.com/embed/zyku2ibsG0M" alt="Video about adding users to Mattermost using AD/LDAP" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>
```

**Review by cwarnermm — CHANGES_REQUESTED** [review]
Thank you, @shikhar13012001! I've provided feedback inline.

**Comment by cwarnermm** [comment]
@shikhar13012001 - Please revert the single change applied to an archived docs source file.

**Comment by shikhar13012001** [comment]
@cwarnermm sorry for delay, I was busy with some personal college stuff, I have reverted the file as requested please review the file let, me know if you need further changes.

**Review by cwarnermm — APPROVED** [review]
Thank you, @shikhar13012001!

**Pull request merged**